queries.sql

```diff
@@ -105,3 +105,9 @@ SELECT s.name AS suggested_specialty, COUNT(*) AS animal_count FROM animals a
 JOIN visits v ON a.id = v.animal_id JOIN vets vt ON v.vet_id = vt.id JOIN vets ms ON ms.name = 'Maisy Smith'
 JOIN specializations sp ON vt.id = sp.vet_id AND a.species_id = sp.species_id JOIN species s ON sp.species_id = s.id
 WHERE ms.id = vt.id GROUP BY s.name ORDER BY animal_count DESC LIMIT 1;
+
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4;
+
+EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;
+
+EXPLAIN ANALYZE SELECT * FROM owners where email = 'owner_18327@mail.com';
```

Comment:
- [ ] 👍 Excellent job with your screenshots! You've provided both the before and after images, which is fantastic. However, I noticed that the execution time in the screenshots for  `vet_id ` and `email ` taken before creating the index is significantly lower than 1 second (1000ms), as mentioned in the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=Depending%20on%20your,bigger%20than%201000ms.). To ensure accuracy, kindly consider running the populate database query a few more times, especially if your machine is particularly fast. This will help capture execution times that are consistently above the 1000ms threshold as required. Keep up the great work! 👍⏱🔍📸

Reply:
@shahadat3669 Thanks. But how do I do to resubmit? Since I'll have to take a different snapshot.